### PR TITLE
ios-webkit-debug-proxy HEAD

### PR DESCRIPTION
**Pull request by artygus**
Add HEAD to ios-webkit-debug-proxy formula

### CODE REVIEW

File: Formula/ios-webkit-debug-proxy.rb [hunk 1]
```diff
@@ -26,4 +28,8 @@ def install
     system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
     system "make", "install"
   end
+
+  test do
+    system "#{bin}/ios_webkit_debug_proxy", "--help"
```
Comment (woodruffw) [inline Formula/ios-webkit-debug-proxy.rb]: Any chance of a more in-depth test here?

Comment (artygus) [inline Formula/ios-webkit-debug-proxy.rb]: @woodruffw I don't think so. The main purpose of the utility is to proxy requests from usbmuxd daemon, which could be either started or not and a running device, or at least iOS Simulator

Comment (woodruffw) [inline Formula/ios-webkit-debug-proxy.rb]: Got it, thanks for the explanation.

**Comment by nijikon** [comment]
You are awesome, thanks for your contribution! :tada:

**Pull request closed without merging**